contributors/design-proposals/service-injection-policy.md

```diff
@@ -0,0 +1,444 @@
+# Service Injection Policy
```

Comment:
Let's rename this "Pod Injection Policy" per our discussions at the f2f last week